Tyk 5.8.10 RNs.mdx

JoanCamosTyk opened the pull request:
### **PR Type**
Documentation


___

### **Description**
- Add 5.8.10 Gateway release notes

- Include compatibility and dependencies matrices

- Note deprecation of explicit policy ID option

- Provide downloads, upgrade, and changelog anchors


___

### Diagram Walkthrough


```mermaid
flowchart LR
  notes["Add 5.8.10 Release Notes"] -- "compatibility matrix" --> compat["Components & 3rd-party deps"]
  notes -- "upgrade guidance" --> upgrade["Upgrade instructions link"]
  notes -- "deprecation" --> deprec["Deprecate policies.allow_explicit_policy_id"]
  notes -- "downloads" --> downloads["Docker, Helm, source links"]
  notes -- "changelog section" --> changelog["Fixed (placeholder)"]
```



<details> <summary><h3> File Walkthrough</h3></summary>

<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>gateway.mdx</strong><dd><code>Introduce Gateway 5.8.10 release notes section</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

developer-support/release-notes/gateway.mdx

<ul><li>Add new "5.8.10 Release Notes" section.<br> <li> Include components compatibility and third-party dependencies tables.<br> <li> Document deprecation of <code>policies.allow_explicit_policy_id</code>.<br> <li> Add downloads, upgrade link, and changelog anchor/structure.</ul>


</details>


  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/1220/files#diff-9095026f67d8773f400fd494ced0d00ea8034dc1ccd14aa0988038ca8f45d0be">+68/-0</a>&nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

</details>

___

alephnull commented:
> So this is a change starting from 2.9.4? Or 2.9.4.1? Or maybe this docs work for people that use 2.9.3?

This would be applicable to 2.9.3. However, there are other bugs plaguing 2.9.3 which are not going to be resolved. Either way, this should be official mount path. Because of the way paths are important to goplugins, I might change the internal paths based on need without impacting docs.

JoanCamosTyk committed:
Update gateway.mdx

JoanCamosTyk committed:
Update gateway.mdx

JoanCamosTyk committed:
Update dashboard.mdx

JoanCamosTyk committed:
Update overview.mdx

JoanCamosTyk committed:
Update gateway.mdx

JoanCamosTyk committed:
Update gateway.mdx

JoanCamosTyk committed:
Update dashboard.mdx

JoanCamosTyk committed:
Update gateway.mdx

JoanCamosTyk committed:
Update dashboard.mdx

ilijabojanovic committed:
Update release date and version numbers in release notes

Updated the release date and modified version numbers for Sync, EDP, and Pump.

ilijabojanovic committed:
Correct release date and update version recommendations

Updated release date and version recommendations for Gateway 5.8.10.

ilijabojanovic committed:
Update release date and Sync version in release notes

Updated the release date for version 5.8.10 and modified the recommended release for Sync.

ilijabojanovic committed:
Merge branch 'production' into JoanCamosTyk-patch-14

ilijabojanovic commented:
/release to main
